runner.go

```diff
@@ -46,7 +47,8 @@ func newRuntime() *Runtime {
 	}
 	r.flags.StringVar(&configFile, "config", "", "Configuration file to use")
 	r.flags.BoolVar(&nonInteractive, "non-interactive", false, "Non-interactive mode (only applies when running the default set of tasks)")
-	r.flags.BoolVar(&listTasks, "list", false, "List all tasks")
+	r.flags.BoolVar(&listTasks, "list", false, "List all default tasks")
+	r.flags.BoolVar(&listAllTasks, "listall", false, "List all tasks, including hidden")
```

Comment:
Good thinking.

Reply:
Nit: should this be `listAll` instead of `listall`?

Reply:
updated